### PR TITLE
version: add error handling

### PR DESCRIPTION
**Pull request by hwdef**
Signed-off-by: hwdef <hwdef97@gmail.com>

### Ⅰ. Describe what this PR did

add error handling

### Ⅱ. Does this pull request fix one issue?

Ref: #952

### CODE REVIEW

File: version/version.go [hunk 1]
```diff
@@ -125,7 +125,9 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, fmt.Sprintf("error encoding JSON: %s", err), http.StatusInternalServerError)
 	} else {
 		w.WriteHeader(http.StatusOK)
-		w.Write(data)
+		if _, err := w.Write(data); err != nil {
+			http.Error(w, fmt.Sprintf("error writing the data to the connection: %s", err), http.StatusInternalServerError)
+		}
```
Comment (allencloud) [inline version/version.go]: How about:
```
if _, err := w.Write(data); err != nil {
    http.Error(w, fmt.Sprintf("error writing the data to the connection: %s", err), http.StatusInternalServerError)
} else {
    w.WriteHeader(http.StatusOK)
}
```

Comment (Starnop) [reply version/version.go]: Agree.

Comment (hwdef) [reply version/version.go]: You are right,
I fixed it.

Comment (allencloud) [reply version/version.go]: If that, you can combine the code like:
```
if err != nil {
    http.Error(w, fmt.Sprintf("error encoding JSON: %s", err), http.StatusInternalServerError)
} else if _, err := w.Write(data); err != nil {
    http.Error(w, fmt.Sprintf("error writing the data to the connection: %s", err), http.StatusInternalServerError)
} else {
    w.WriteHeader(http.StatusOK)
}
```
And in this way, you could reduce the ident? right? WDYT

Comment (hwdef) [reply version/version.go]: Thank you！
You are very good at the code style of golang.
I will change it.

**Comment by allencloud** [comment]
LGTM，code format is the way you talk, communicate. We are always bringing convenience when delivering code. Thanks for your update. Wish to write strict and friendly code together. @hwdef

**Pull request merged**